.travis.yml

```diff
@@ -17,8 +17,8 @@ install: source build_tools/travis/install.sh
 
 script:
   - if [ ${COVERAGE} == "true" ];
-    then nosetests --with-coverage --cover-package=skopt --with-timer --timer-top-n 10; else
-    nosetests --with-timer --timer-top-n 10;
+    then travis_wait 25 pytest --cov=skopt --durations=10; else
+    travis_wait 25 pytest --durations=10;
```

Comment:
In Python 2, one of the tests in `skopt/tests/test_gp_opt.py` took longer than 10 mins to finish. I added this 25 minute buffer, so that Travis CI does not error out when a test is taking too long.